infra/gcp/clusters/projects/k8s-infra-public-pii/main.tf

```
@@ -0,0 +1,117 @@
+
+
+/**
+ * Copyright 2021 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  project_id  = "k8s-infra-public-pii"
+  bucket_name = "k8s-infra-artifacts-gcslogs"
+}
+
+
+data "google_billing_account" "account" {
+  billing_account = "018801-93540E-22A20E"
+}
+
+data "google_organization" "org" {
+  domain = "kubernetes.io"
+}
+
+
+resource "google_project" "project" {
+  name            = local.project_id
+  project_id      = local.project_id
+  org_id          = data.google_organization.org.org_id
+  billing_account = data.google_billing_account.account.id
+}
+
+resource "google_project_service" "project" {
+  project = google_project.project.id
+
+  for_each = toset([
+    "bigquery.googleapis.com",
+    "logging.googleapis.com",
+    "storage-component.googleapis.com"
+  ])
+
+  service = each.key
+}
+
+
+// BigQuery dataset for audit logs
+resource "google_bigquery_dataset" "audit-logs-gcs" {
+  dataset_id                  = local.bucket_name
+  friendly_name               = local.bucket_name
+  delete_contents_on_destroy  = true
```

```suggestion
  delete_contents_on_destroy  = false
```
I mean maybe we should treat this as an ephemeral cache of data that can be arbitrarily destroyed at any time? But I personally err on the side of "data is sacred"

---

Fixed